tests/smoke/test_orchestrator_smoke.py

```diff
@@ -40,3 +40,26 @@ def test_orchestrator_different_epoch(
     assert results.nonwear_epoch is not None
     assert results.sleep_windows_epoch is not None
     assert results.physical_activity_levels is not None
+
+
+@pytest.mark.parametrize(
+    "file_name", [pathlib.Path("test_output.csv"), pathlib.Path("test_output.parquet")]
+)
+def test_happy_path_range_criteria(
+    file_name: pathlib.Path, tmp_path: pathlib.Path, sample_data_gt3x: pathlib.Path
+) -> None:
+    """Happy path for orchestrator."""
+    settings_range_criteria = config.Settings(RANGE_CRITERIA=0.1)
+    results = orchestrator.run(
+        input=sample_data_gt3x,
+        output=tmp_path / file_name,
+        settings=settings_range_criteria,
+    )
+
+    assert (tmp_path / file_name).exists()
+    assert isinstance(results, orchestrator.Results)
+    assert results.enmo is not None
+    assert results.anglez is not None
+    assert results.nonwear_epoch is not None
+    assert results.sleep_windows_epoch is not None
+    assert results.physical_activity_levels is not None
```

Comment:
Just realized there was one change I forgot to make in my PR here, we decided that these assert checks should actually just directly check that they're models.Measurement.